src/features/Search/components/CharacterCoverCard/CharacterCoverCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo, VFC } from 'react';
```

Comment:
Maybe, we should put components specific for `character` page inside `pages/CharacterSearchPage/components` (and for `vn` page components too)? At least, that's how we usually do on `Angular` projects.